server.mjs

```diff
@@ -0,0 +1,88 @@
+// setup web server
+import express from 'express';
+import path from 'path';
+import fs from 'fs';
+import config from "./config.json";
+import log from "./public/js/logger.mjs";
+
+// Constants
+const PORT = config[process.env['NODE_ENV']].port;
+const HOST = config[process.env['NODE_ENV']].host;
```

Comment:
Make server considerate of running environment to enable expansion into cloud environments.